recipes/lagraph/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "8Aug2020" %}
+
+package:
+  name: lagraph
+  version: {{ version }}
+
+source:
+  url: https://github.com/GraphBLAS/LAGraph/archive/{{ version }}.tar.gz
+  sha256: e7a05decd21d2f209ca24046447bff7a9eefe54b071eca5f9fedf6d468bda58c
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - m4
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+  host:
+    - graphblas
+  run:
+    - graphblas
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/LAGraph.h
+    - test -f ${PREFIX}/lib/liblagraph${SHLIB_EXT}
+    - ! test -f ${PREFIX}/lib/liblagraph.a
+
+about:
+  home: https://github.com/GraphBLAS/LAGraph
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: This is a library plus a test harness for collecting algorithms that use the GraphBLAS
+
+extra:
+  recipe-maintainers:
+    - jim22k
```

Comment:
@jim22k are you willing to be a maintainer?

Reply:
Yes